packages/runner-shared/src/header/index.tsx

```diff
@@ -0,0 +1,242 @@
+import cs from 'classnames'
+
+import { action, computed, observable } from 'mobx'
+import { observer } from 'mobx-react'
+import React, { Component, createRef } from 'react'
+import Tooltip from '@cypress/react-tooltip'
+// @ts-ignore - yes it does!
+import { $ } from '@packages/driver'
```

Comment:
TS thinks that `$` is not exported from `@packages/driver`, but actually it is.

Reply:
I think you could update `packages/driver/index.d.ts` to correctly reflect the export

Reply:
Done - did not realize this was typed with a `.d.ts` file.